AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -71,11 +71,13 @@ - (void)setAuthToken:(NSString *)authToken withAccountId:(NSString *)accountId {
   }
 }
 
-- (void)clearAuthToken {
+- (BOOL)clearAuthToken {
   NSArray *synchronizedDelegates;
   BOOL clearedExistingUser = NO;
   @synchronized(self) {
-    if (self.authToken != nil && self.homeAccountId != nil) {
+    if (!self.authToken) {
+      return NO;
```

Comment:
Add a test case when `clearAuthToken:` returns `NO`.

Reply:
It's already covered in https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1331/files#diff-1f576a0979891ad20bae04d644bbc7eeR91. Ignore my comment.